orion/1.12/mastermind.yml

```diff
@@ -0,0 +1,32 @@
+name: orion
+version: 1.12
+description: MongoDB Replicaset to work with Fiware IoT Services
```

Comment:
description does not match name

Reply:
@taliaga Oups. I was using another file as template, forgot to change this